package/CHANGELOG

```diff
@@ -30,6 +32,7 @@ Fixes
     (e.g. bonds, angles) (PR #3779).
 
 Enhancements
+  * Quality of life updates to libdcd Cython (Issue #3882, PR #3888)
```

Comment:
Be more specific.

Reply:
If it improves performance then say that. If it had any user facing consequences then definitely say so. If it’s code modernization then we can list it (but we don’t always have been consistent iirc - @IAlibay ?.

Reply:
Pretty much, we list it if there's a direct user impact (we have historically taken a poor stance towards acknowledging QoL / maintenance work unfortunately, that's something to be revisted)